DEPS

```diff
@@ -3,7 +3,7 @@ use_relative_paths = True
 deps = {
   "vendor/ad-block": "https://github.com/brave/ad-block.git@11ab48c4b2bd5355e789d4c87005ab2c14eecc09",
   "vendor/autoplay-whitelist": "https://github.com/brave/autoplay-whitelist.git@458053a3c95b403cbe0872f289a2aafa106ee9d8",
-  "vendor/extension-whitelist": "https://github.com/brave/extension-whitelist.git@463e5e4e06e0ca84927176e8c72f6076ae9b6829",
+  "vendor/extension-whitelist": "https://github.com/brave/extension-whitelist.git@c9c442343ff4711726e4792569bbac8ff3d980b9",
```

Comment:
This should be updated after merging https://github.com/brave/extension-whitelist/pull/5

Reply:
merged!

Reply:
Done.